src/Raven.Client/Data/DatabaseStatistics.cs

```diff
@@ -79,6 +79,8 @@ public class IndexInformation
         public IndexType Type { get; set; }
 
         public DateTime? LastIndexingTime { get; set; }
+
+        public string OutputReduceToCollection { get; set; }
```

Comment:
I think that a more suitable place would be to move it to /index/stats and remove it from here